fe/fe-core/src/main/java/com/starrocks/sql/ast/AlterStorageVolumeClause.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.sql.ast;
+
+import com.google.common.collect.Maps;
+import com.starrocks.analysis.ParseNode;
+import com.starrocks.sql.parser.NodePosition;
+
+import java.util.Map;
+
+public abstract class AlterStorageVolumeClause implements ParseNode {
+    public enum AlterOpType {
+        ALTER_COMMENT,
```

Comment:
```suggestion
        MODIFY_COMMENT,
```

Reply:
It may be better the keep the same naming style with create table.